client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -260,6 +266,30 @@ class PodcastingDetails extends Component {
 						onAddTermSuccess={ this.onCategorySelected }
 						height={ 200 }
 					/>
+					{ isDefaultCategorySelected && (
+						<Notice isCompact status="is-info">
+							<p>
+								{ translate(
+									'Using this category (the {{strong}}default category{{/strong}} for the site) is not recommended.',
```

Comment:
This feels a little redundant. It could simply be "Using the site's **default category** is not recommended."

Reply:
I wrote it this way because there is not a clear relationship between "Uncategorized" and "default category".  But it looks like we will probably not be going with this as written anyway so I'll wait until we decide on an approach here.